### PR TITLE
Adding support for Syscoin

### PR DESCRIPTION
**Pull request by tomatoskittles**

### CODE REVIEW

File: _data/chains/eip155-5800.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Syscoin Testnet",
+  "chain": "SYS",
+  "network": "testnet",
+  "rpc": [
+    "https://tanenbaum.io/api",
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json]: https://tanenbaum.io/api/eth-rpc, no ws

File: _data/chains/eip155-57.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Syscoin Mainnet",
+  "chain": "SYS",
+  "network": "mainnet",
+  "rpc": [
+    "https://tanenbaum.io/api",
```
Comment (sidhujag) [inline _data/chains/eip155-57.json]: https://nevm.syscoin.org/api/eth-rpc, https://web3.syscoin.org, wss://web3.syscoin.org

File: _data/chains/eip155-57.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "symbol": "SYS",
+    "decimals": 18
+  },
+  "infoURL": "https://www.syscoin.org",
+  "shortName": "sys",
+  "chainId": 57,
+  "networkId": 57,
+  "explorers": [{
+    "name": "syscoin explorer",
+    "url": "https://tanenbaum.io",
```
Comment (sidhujag) [inline _data/chains/eip155-57.json]: nevm.syscoin.org not tanenbaum

File: _data/chains/eip155-5800.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://www.syscoin.org",
+  "shortName": "sys",
+  "chainId": 5800,
+  "networkId": 5800,
+  "explorers": [{
+    "name": "syscoin explorer",
+    "url": "https://syscoin.readme.io/",
+    "standard": "EIP3091???"
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json]: it is not eip3091 compliant, dbl check this pls

File: _data/chains/eip155-57.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    "standard": "EIP3091???"
```
Comment (sidhujag) [inline _data/chains/eip155-57.json]: not eip3091 compliant, dbl check

File: _data/chains/eip155-5800.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Syscoin Testnet",
+  "chain": "SYS",
+  "network": "testnet",
+  "rpc": [
+    "https://tanenbaum.io/api",
+    "wss://???.io/ws"
+  ],
+  "faucets": ["https://faucet.tanenbaum.io"]
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json]: faucet.syscoin.org (not tanenbaum.io)

File: _data/chains/eip155-5800.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    "symbol": "SYS",
+    "decimals": 18
+  },
+  "infoURL": "https://www.syscoin.org",
+  "shortName": "sys",
+  "chainId": 5800,
+  "networkId": 5800,
+  "explorers": [{
+    "name": "syscoin explorer",
+    "url": "https://syscoin.readme.io/",
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json]: tanenbaum.io not readme.io

File: _data/chains/eip155-5800.json [hunk 8]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Syscoin Testnet",
+  "chain": "SYS",
+  "network": "testnet",
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json:4]: tanenbaum not testnet

File: _data/chains/eip155-5800.json [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    "wss://???.io/ws"
+  ],
+  "faucets": ["https://faucet.tanenbaum.io"]
+  "nativeCurrency": {
+    "name": "Syscoin",
+    "symbol": "SYS",
+    "decimals": 18
+  },
+  "infoURL": "https://www.syscoin.org",
+  "shortName": "sys",
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json]: tsys not sys

**Review by sidhujag — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-5800.json [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+  "chain": "SYS",
+  "network": "testnet",
+  "rpc": [
+    "https://tanenbaum.io/api",
+    "wss://???.io/ws"
+  ],
+  "faucets": ["https://faucet.tanenbaum.io"]
+  "nativeCurrency": {
+    "name": "Syscoin",
+    "symbol": "SYS",
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json]: tSYS not SYS

File: _data/chains/eip155-5800.json [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "Syscoin Testnet",
+  "chain": "SYS",
+  "network": "testnet",
+  "rpc": [
+    "https://tanenbaum.io/api",
+    "wss://???.io/ws"
+  ],
+  "faucets": ["https://faucet.tanenbaum.io"]
+  "nativeCurrency": {
+    "name": "Syscoin",
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json]: Testnet Syscoin not Syscoin

File: _data/chains/eip155-5800.json [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+  "infoURL": "https://www.syscoin.org",
+  "shortName": "sys",
+  "chainId": 5800,
+  "networkId": 5800,
+  "explorers": [{
+    "name": "syscoin explorer",
+    "url": "https://syscoin.readme.io/",
+    "standard": "EIP3091???"
+  }]
+}
```
Comment (sidhujag) [inline _data/chains/eip155-5800.json:24]: infoURL missing?

File: _data/chains/eip155-57.json [hunk 13]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Syscoin Mainnet",
+  "chain": "SYS",
+  "network": "mainnet",
```
Comment (sidhujag) [inline _data/chains/eip155-57.json:4]: Syscoin not mainnet

**Comment by sidhujag** [comment]
looks good

**Comment by ghost** [comment]
@ #

> 
Duplicate of #

**Comment by tomatoskittles** [comment]
> @ #
> 
> > 
> 
> Duplicate of #

@khai911013 I'm not sure what this comment means... Can you elaborate? I want to delete this PR, as it is no longer needed. All of the changes have been merged to the upstream, via another branch and PR.

**Pull request closed without merging**